modules/source-formatter.properties

```diff
@@ -25,6 +25,7 @@
         com.liferay:com.liferay.ant.bnd:3.2.10,\
         com.liferay:com.liferay.whip:1.0.3,\
         com.liferay:com.sun.el.javax.el:3.0.1.LIFERAY-PATCHED-1,\
+        com.liferay:org.apache.axis:1.4.LIFERAY-PATCHED-7,\
         com.liferay:org.apache.commons.logging:1.2.LIFERAY-PATCHED-2,\
         com.liferay:org.apache.felix.configadmin:1.9.8.LIFERAY-PATCHED-6,\
         com.liferay:org.apache.felix.gogo.runtime:1.1.2.LIFERAY-PATCHED-1,\
```

Comment:
@marianoalvarosaiz, when adding new rule, if using liferay path lib, I think we also add the related rule to `source.check.GradleDependencyArtifactsCheck.renameArtifacts` to prevent directly using the third party(`compileOnly group: "axis", name: "axis", version: "1.4"`).